src/Sulu/Component/Rest/ListBuilder/Doctrine/DoctrineListBuilder.php

```diff
@@ -101,7 +101,7 @@ public function count()
         } elseif ($numResults == 1) {
             $result = array_values($result[0]);
 
-            return $result[0];
+            return (int) $result[0];
```

Comment:
I also preferred `intval` but when you read about vs `typecast (int)` the typecast seems to be the better solution aslong as you don't need the base parameter. maybe we can add this to the style.ci to use the `(int)` instead of `intval`.

so :+1:  for `(int)`